jis-plugins/src/main/java/pl/edu/uj/JImageStream/filters/color/BlueFilter.java

```diff
@@ -1,4 +1,4 @@
-package pl.edu.uj.JImageStream.filters;
+package pl.edu.uj.JImageStream.filters.color;
 
 import pl.edu.uj.JImageStream.api.core.Filter;
 import pl.edu.uj.JImageStream.model.Pixel;
```

Comment:
weren't we supposed to change behavior of blue/red/green filters to filter out chosen color instead of leaving only one chosen?